### PR TITLE
Gene Finder Project

### PR DESCRIPTION
**Pull request by vickymmcd**
Turning in Week 1 of project

### CODE REVIEW

File: gene_finder.py [hunk 1]
```diff
@@ -25,13 +26,27 @@ def get_complement(nucleotide):
 
         nucleotide: a nucleotide (A, C, G, or T) represented as a string
         returns: the complementary nucleotide
+
+        added doctest for input of empty string and random letter
     >>> get_complement('A')
     'T'
     >>> get_complement('C')
     'G'
+    >>> get_complement('')
```
Comment (segerphilip) [inline gene_finder.py]: Good extra tests and edge-case checks!

File: gene_finder.py [hunk 2]
```diff
@@ -40,13 +55,21 @@ def get_reverse_complement(dna):
...
+    ''
     """
-    # TODO: implement this
-    pass
+    complement = ''
+    i = len(dna)-1
+    while i >= 0:
+        complement = complement + get_complement(dna[i])
+        i = i-1
+    return complement
```
Comment (segerphilip) [inline gene_finder.py]: This works. Another option is `dna = dna[::-1]` which reverses and saves as dna
